src/server/pfs/cmds/cmds_integration_test.go

```diff
@@ -0,0 +1,130 @@
+//go:build k8s
+
+package cmds
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff"
```

Comment:
This is the wrong backoff library.  I think you wanted src/internal/backoff.

Reply:
Turns out that is the one the test was using before. The code didn't compile with the internal library either :/ 
I tried to adopt the internal library with a few tweaks, since I assume we prefer internal over external libraries. It's small but, I'd appreciate it if you could sanity check this since I really haven't worked with go Contexts before.